packages/katello/katello/service-wait

```diff
@@ -88,15 +72,6 @@ after_start() {
   esac
 }
 
-before_stop() {
-  case "$SERVICE" in
-    tomcat6|tomcat)
```

Comment:
I don't know the exact impact of this, but you're probably right that this is no longer an issue with systemd.